app/routes/news._index.mdx

```diff
@@ -26,6 +26,34 @@ import { Anchor } from '~/components/Anchor'
 
 ## 2024
 
+    <CollectionItem
+      className="maxw-none"
+      variantComponent={
+        <CollectionCalendarDate datetime={'May 20, 2024'} />
+      }
+    >
+      <CollectionHeading headingLevel="h3">
+        <Anchor>Einstein Probe Notices, Astro Flavored Markdown for Circulars, and Notices Schema v4.0.0</Anchor>
+      </CollectionHeading>
+      <CollectionDescription>
+
+        - **Einstein Probe** new GCN notice type is now available in JSON format over [Kafka](/quickstart), but are not yet available via email.  They are not available via GCN Classic.
+          - Einstein Probe (EP) Summary: EP is a mission of the [Chinese Academy of Sciences (CAS)](https://ep.bao.ac.cn/ep/), in collaboration with the [European Space Agency (ESA)](https://www.cosmos.esa.int/web/einstein-probe/mission) and the [Max Planck Institute for Extraterrestrial Physics (MPE)](https://www.mpe.mpg.de/7867826/EinsteinProbe), dedicated to time-domain high-energy astrophysics. Its primary goals are to discover high-energy transients and monitor variable objects.
+            - Occurance Rate: The EP notices occur at a rate of ~100 times per year.
+            - Time Delay: The post-event processing and communication of alert is distributed to GCN within ~60 seconds of detection.
+            - Notice Type: The [new WXT notice type (`gcn.notices.einstein_probe.wxt.alert`)](/missions/einstein_probe) provides trigger information including localization.
+            - Notice Schema and Examples: The JSON schema defining the notice format and examples can be viewed in the [Schema Browser](/docs/schema) or the GCN [schema repository](https://github.com/nasa-gcn/gcn-schema).
+
+        - **Astro-Flavored Markdown** of GCN Circulars provides an enhanced experience in using our GCN Circulars submission web form.  It now optionally utilizes a custom form of Markdown, a lightweight markup language.  It allows embedding of data via tables, enriched text formatting, enhancing the already implemented automatic cross-referencing to other Circulars and common astronomy archives. [Documentation on Astro Flavored Markdown](/docs/circulars/markdown) is available.
```

Comment:
- It's spelled "Astro Flavored Markdown", no hyphen, just like "GitHub Flavored Markdown".
- Remove double spaces between sentences.

```suggestion
        - **Astro Flavored Markdown** of GCN Circulars provides an enhanced experience in using our GCN Circulars submission web form. It now optionally utilizes a custom form of Markdown, a lightweight markup language.  It allows embedding of data via tables, enriched text formatting, enhancing the already implemented automatic cross-referencing to other Circulars and common astronomy archives. [Documentation on Astro Flavored Markdown](/docs/circulars/markdown) is available.
```